README.md

```diff
@@ -4,31 +4,25 @@ Midtrans Drupal 8/9 Commerce Payment Gateway Module
 Midtrans :heart: Drupal 8/9!
 Let your Drupal Commerce 2 store integrated with Midtrans payment gateway.
 
-### Description
+#### Description
 This is the official Midtrans extension for the Drupal Commerce 2 E-commerce platform.
 
-### Version
+#### Version
 2.1.0
 (for Drupal v8.x and Drupal v9.x)
 
-### Requirements
-The following plugin is tested under following environment:
+#### Requirements
+The following module is tested under following environment:
 
 * PHP v5.6.x or greater
 * MySQL version 5.0 or greater
+* [Midtrans PHP Library](https://github.com/Midtrans/midtrans-php)
 * [Drupal v8.x or greater](https://www.drupal.org/project/drupal)
 * [Drupal Commerce 8.x-2.xx ](http://www.drupal.org/project/commerce)
 
-#### Composer Installation
-If you are using [Composer](https://getcomposer.org), you can install via composer CLI
-run: `composer require drupal/midtrans_commerce`  on your terminal.
-
-#### Manual Instalation
-The manual installation method involves downloading our feature-rich plugin and uploading it to your webserver via your favourite FTP application.
-
-1. Download the plugin file to your computer and unzip it, rename folder to ``commerce_midtrans``.
-2. Using an FTP program, or your hosting control panel, upload the unzipped plugin folder to your Drupal modules installation's ``[Drupal folder]/modules/contrib/`` directory.
```

Comment:
hmm so the manual folder based install will no longer works, because the dependency is Midtrans php via composer, which is the only external dependency. Then I think it is better to: please include the Midtrans PHP as a built in folder in this repo, instead of compser dependency. 

So that no external dependency & manual folder based install will work, there's usually no need to update Midtrans PHP that often anyway.

Reply:
make sense, ok, will include the Midtrans PHP as a built-in folder in this repo

Reply:
yes thanks. This is to make sure we have alternatives to install beside from Drupal composer repo. Because sometimes they can shut down our plugin at any time. Just like when our WC plugin is removed from store, because WP have new security template to follow.

Reply:
add new commit, already tested and run well